src/us/kbase/narrativemethodstore/test/FullServerTest.java

```diff
@@ -1,5 +1,7 @@
 package us.kbase.narrativemethodstore.test;
 
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
```

Comment:
does that module not get loaded by `import static org.junit.Assert.*`?

Reply:
Wow, I didn't even notice the global import. I switched it to just importing the names that are needed.